pxr/imaging/plugin/hdRpr/mesh.cpp

```diff
@@ -474,7 +474,8 @@ void HdRprMesh::Sync(HdSceneDelegate* sceneDelegate,
                     }
                 } else {
                     updateTransform = false;
-                    GfMatrix4d meshTransform(m_transform);
+                    // TODO: handle instancer animated transforms
+                    GfMatrix4d meshTransform(m_transformSamples.Resample((m_transformSamples.times[0] + m_transformSamples.times[m_transformSamples.count - 1]) * 0.5f));
```

Comment:
The instancer transform animation is much more complex. So I will do it in a separate PR